src/py42/clients/_archiveaccess/__init__.py

```diff
@@ -0,0 +1,165 @@
+import posixpath
```

Comment:
I know a lot of these files are all green from the refactor, but they could use an overall review anyway, so I don't mind.

Reviewing this PR may take some time...